### ISSUE TITLE
Has there been a change in how self-closing tags are validated?

### ISSUE THREAD

**Issue by FrankPal** [validator/validator#1430]
URL being validated or code to reproduce error:
http://www.ussupernet.com/

```
Self-closing tag syntax in text/html documents is [widely discouraged](https://google.github.io/styleguide/htmlcssguide.html#Document_Type);
```

This page has validated with no errors or warnings for years.  It uses `<!DOCTYPE html>`
Starting today, tags such as these create warnings:
```
<meta name="keywords" content="web development,web hosting,consulting,internet consulting,domain name registration,westerly,internet,hosting" />
<br/>
<link rel="shortcut icon" href="CapsShieldIcon.ico" type="image/x-icon" />
<img src="HomePhotos/CentralPark1860x750.jpg" width="1860" height="750" alt="Central Park" title="Central Park"  class="img-responsive img-rounded center-block" />
```

Is this a bug or have things changed?

**Comment by phpfui** [comment]
I just bumped into this today as well via a complete fluke.

I adjusted by source to comply with these requirements, as the best I can tell, they are correct.

I also found it was validating <input type="date"> values.  0000-00-00 is not considered valid.

My issue is that there as not been a new release in over 2 years.  I test against the June 30, 2020 release, and these errors are not flagged.

Ideally we need a new release.  I don't see a vnu.jar file to download for the pre-release.

So all good, except for a new release!

**Comment by sideshowbarker** [comment]
@FrankPal It’s not a bug but instead it’s an intentional change that I made yesterday.

If you prefer to keep the document as-is rather than changing those tags to, e.g., `<br>` instead of `<br/>`, then you can use one of the checker’s message-filtering mechanisms to persistently filter out the _“Self-closing tag syntax in text/html documents is widely discouraged”_ warning messages.

For example, you can use the <kbd>Message Filtering</kbd> button in the checker UI:

![Screen-Recording-2022-09-28-at-1(1)](https://user-images.githubusercontent.com/194984/192681214-b9be680f-9ef0-4e7c-a618-e7a49e32095b.gif)

**Comment by sideshowbarker** [comment]
@phpfui Thanks for the bump — I will try to get a new release out soon

**Comment by phpfui** [comment]
Awesome! This is a very valuable tool.  I used it all the time in unit test to make sure my HTML is valid.  All my sites validate 100% now with no errors or warnings.  Nice stuff.

It is amazing how many (most) sites have bad HTML.  UGH!

**Comment by FrankPal** [comment]
I agree - that you've developed a very valuable tool here.  I use it daily to check my work and keep things clean.  For years, I have been thinking that these self closing tags, such as:
<img src="HomePhotos/CentralPark1860x750.jpg" width="1860" height="750" alt="Central Park" title="Central Park"  class="img-responsive img-rounded center-block" /> and <br/>
